python/nano/test/pytorch/tests/inference/test_ipex_jit_inference.py

```diff
@@ -577,6 +578,22 @@ def test_ipex_jit_inference_stable_diffusion_unet(self):
         new_model(image_latents, torch.Tensor([980]).long(), encoder_hidden_states)
         new_model(image_latents2, torch.Tensor([980]).long(), encoder_hidden_states2)
 
+    @pytest.mark.skipif(compare_version("torch", operator.lt, "1.13"),
+                        reason="jit_int8 is only supported when torch>=1.13")
+    def test_jit_int8(self):
+        model = ResNet18(10, pretrained=False, include_top=False, freeze=True)
+        loader = create_data_loader(data_dir, batch_size, num_workers, data_transform)
+        jit_int8_model = InferenceOptimizer.quantize(model,
+                                                     calib_dataloader=loader,
+                                                     accelerator="jit",
+                                                     precision="int8")
+        input_sample = next(iter(loader))[0]
+        with InferenceOptimizer.get_context(jit_int8_model):
+            output1 = jit_int8_model(input_sample)
+        with InferenceOptimizer.get_context(model):
+            output2 = model(input_sample)
+        np.testing.assert_allclose(output1, output2, atol=2e-1)
```

Comment:
I wonder should we test this with a really large `atol`, since we can not guarantee the int8 accuracy right? (but if it keeps working, I am fine with it)

Reply:
In my test, the atol may achieve about 0.1 sometimes, so I choose 0.2